src/bin/deku_node.ml

```diff
@@ -15,6 +15,82 @@ let exits =
 let man = [`S Manpage.s_bugs; `P "Email bug reports to <contact@marigold.dev>."]
 
 (* TODO: several functions copied from deku-cli. Refactor. *)
+type json_option =
+  | No
+  | To_stdout
+  | To_stderr
```

Comment:
The problem with these options is that they include messages from the bridge as strings, whereas it should probably be included as JSON too. As I said in another PR, I don't exactly know how the JSON logs are subsequently used so it's hard to know if it's relevant or not.

Moreover, and keeping the previous comment in mind, I believe the node identity should be included in the JSON output. Although each node most likely runs in its own space and has its own log file/stdout/journal/whatever, I think it would make it easier to analyze the logs.

Reply:
These are excellent points that I don't want to address here :sweat_smile: